plugins/heartbeats/scripts/heartbeats.py

```
@@ -0,0 +1,403 @@
+        if tokens:
+            names.append(tokens[0])
+    return names
+
+
+def _removed_unit_count(live_region, block):
+    """How many units the live region names that the new block no longer does."""
+    surviving = set(_unit_header_names(block))
+    return sum(1 for name in _unit_header_names(live_region) if name not in surviving)
+
+
+def _run_against_crontab(args, units, block):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_run_against_crontab`

(ANN202)